scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/mocks/InMemoryResultsCellRepository.java

```diff
@@ -1,19 +1,24 @@
 package gov.nasa.jpl.aerie.scheduler.server.mocks;
 
+import java.util.Optional;
 import gov.nasa.jpl.aerie.scheduler.server.ResultsProtocol;
 import gov.nasa.jpl.aerie.scheduler.server.models.SpecificationId;
 import gov.nasa.jpl.aerie.scheduler.server.remotes.ResultsCellRepository;
 import gov.nasa.jpl.aerie.scheduler.server.remotes.SpecificationRepository;
 
-import java.util.Optional;
-
 public record InMemoryResultsCellRepository(SpecificationRepository specificationRepository) implements ResultsCellRepository {
   @Override
   public ResultsProtocol.OwnerRole allocate(final SpecificationId specificationId)
   {
     throw new UnsupportedOperationException(); // TODO stubbed method must be implemented
   }
 
+  @Override
+  public Optional<ResultsProtocol.OwnerRole> claim(SpecificationId specificationId)
+  {
+    throw new UnsupportedOperationException(); // TODO stubbed method must be implemented
+  }
+
```

Comment:
Does this have to be implemented for this PR or is this future work?

Reply:
Future work :) On the Merlin side of things this in-memory results cell repo. exists to help with testing without a true PG DB up and running. We can eventually support this for the scheduler as well.